run-tests-with-build-drools-parent/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>run-tests-with-build</artifactId>
+    <groupId>org.kie</groupId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>pom</packaging>
+
+  <artifactId>run-tests-with-build-drools-parent</artifactId>
+  <properties>
+    <!-- We can not skip src/main/java module compilation -->
+    <maven.main.skip>false</maven.main.skip>
```

Comment:
I think this is difference to previous state of things in drools upstream tests. Not skipping means we don't test productized JARs of this maven artifact, but just its dependencies. This will be problem for drools core modules I think.
There's a plan to fix this through: kiegroup/unpack-build-maven-plugin#1 - that change would be applicable also to rest of these *-parent modules, fixing this testing gap we have.
I could prioritize the work on unpack-build-maven-plugin to unblock here.